src/messages.json

```diff
@@ -534,11 +534,10 @@
         "WorldSelectorButton.label": "Scene Background selector"
     },
     "fr": {
-        "ActionPanel.action.delete": "supprimer {stepNumber} {stepName}",
-        "ActionPanel.action.moveToNextStep": "remplacer l'étape {stepNumber} {stepName} {nextStepInfo}",
-        "ActionPanel.action.moveToPreviousStep": "remplacer l'étape {stepNumber} {stepName} {previousStepInfo}",
-        "ActionPanel.action.replace": "remplacer l'étape {stepNumber} {stepName} {selectedCommandName}",
-        "ActionPanel.selectedCommandName": "{selectedCommandName} avec action sélectionner",
+        "ActionPanel.action.delete": "supprimer {stepNumber} {stepActionName}",
+        "ActionPanel.action.moveToNextStep": "remplacer l'étape {stepNumber} {stepActionName} {nextStepInfo}",
+        "ActionPanel.action.moveToPreviousStep": "remplacer l'étape {stepNumber} {stepActionName} {previousStepInfo}",
+        "ActionPanel.action.replace": "remplacer l'étape {stepNumber} {stepActionName} {selectedActionName}",
```

Comment:
Thanks @sbates-idrc for reviewing the changes, I didn't update French messages yet because we don't have French translations yet. I think it makes sense to update them all when we get the translations.